python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -323,17 +288,6 @@ def trackandRelease(self):
 
                  flush_thread = threading.Thread(target=self._flush, args=())
                  flush_thread.run()
-
-                 for obj in self.dirty_objs.itervalues():
-
-                    if self.shouldReleaseRun is False:
-                        break
-
-                    _args = ([obj])
-                    release_thread = threading.Thread(target=self._release_lock, args=_args)
```

Comment:
I think in the interest of stability I'll opt for keep this removed here and I'll add a quick 2 or 3 line PR for the prep repository which is less invasive than adding yet another thread of repo actions.